### PR TITLE
Added link to the commute planner

### PR DESCRIPTION
**Pull request by OfficialFreak**
- [x] I made sure that the arrays in `routes.php` are still alphabetically sorted. :sparkles:

### CODE REVIEW

File: redirect/routes.php [hunk 1]
```diff
@@ -871,6 +875,7 @@ class Route {
             'app',
             'artemis',
             'c',
+            'commuteplanner',
             'c2',
```
Comment (joschahenningsen) [inline redirect/routes.php:879]:
```suggestion
            'c2',
            'commuteplanner',
```

**Review by joschahenningsen — APPROVED** [review]
🍻

**Comment by OfficialFreak** [comment]
@joschahenningsen It seems like the content on the main site hasn't updated

**Comment by joschahenningsen** [comment]
@CommanderStorm any hints here?

**Comment by CommanderStorm** [comment]
TUM.sexy currently does not have continuous deployment. Any deployments need to be triggered by clicking "restart"

image

**Comment by CommanderStorm** [comment]
@joschahenningsen we should definitvely meet sometime and build the new deployment infrastructure.
My last exam is on the 23.2.

**Comment by joschahenningsen** [comment]
Definitely, yes. In the meantime, can we trigger a manual deployment?

**Comment by CommanderStorm** [comment]
Do you really think I opened said popup just to make a screenshot?

**Pull request merged**